karma.conf.babel.js

```diff
@@ -91,12 +91,8 @@ export default (karmaConfig) => {
       'test/tests.bundle.js': ['webpack'],
     },
     webpack: {
+      ...webpackConfig,
```

Comment:
This is now spreading keys that aren't applicable to karma's webpack config (`name`, `mode`, `target`, `output`, etc).  I think we should stick with explicitly passing only the keys we want as it was before.

Reply:
Now we have only `webpack.config.karma.js`, so I cleaned up it more and removed `name`, `output`, `target` from there.

`mode` is required setting for Webpack 4, otherwise it will run in production mode.